src/Sulu/Bundle/MediaBundle/Media/FormatLoader/XmlFormatLoader10.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatLoader;
+
+use Sulu\Bundle\MediaBundle\Media\FormatLoader\Exception\MissingScaleDimensionException;
+use Symfony\Component\Config\FileLocatorInterface;
+
+/**
+ * Class XmlFormatLoader for the version 1.1 of the image-formats.
+ *
+ * @deprecated
+ */
+class XmlFormatLoader10 extends BaseXmlFormatLoader
+{
+    const SCHEMA_URI = 'http://schemas.sulu.io/media/formats-1.0.xsd';
+
+    const SCHEME_PATH = '/schema/formats/formats-1.0.xsd';
+
+    public function __construct(FileLocatorInterface $locator)
+    {
+        parent::__construct($locator);
+        @trigger_error(
```

Comment:
That's probably the wrong place, because it will be called every single time. Please move that to the `load` function.